app/interactors/add_movie_to_round.rb

```diff
@@ -0,0 +1,28 @@
+require 'hanami/interactor'
+
+class AddMovieToRound
+  include Hanami::Interactor
+
+  attr_reader :round, :tmdb_id
+
+  def initialize(round:, tmdb_id:)
```

Comment:
Two things I would do differently here:

1. Even though I used to be a proponent of passing the execution parameters on `initialize`, the documentation for Hanami's Interactors is explicit in passing them in the `#call`, which may make more sense, actually;
2. … because `initialize` should be used to inject dependencies. Namely, in this case, it's a big help to inject an instance of `ImportTmdbMovie`. We can set it to `ImportTmdbMovie.new`, so it can be an optional parameter, omitted in typical use-cases, but then inject a special version, with a stubbed API client, for testing.

Reply:
Nice input. Refactored!